src/main/java/ru/mail/polis/homework/objects/RepeatingCharacters.java

```diff
@@ -28,6 +28,11 @@ public static Pair<Character, Integer> getMaxRepeatingCharacters(String str) {
                 if (currLength > maxLength) {
                     maxLength = currLength;
                     maxRepSymbol = currSymbol;
+                    if (maxLength >= arrStr.length / 2 & (i + 1) < arrStr.length) {
+                        if (!(arrStr[i + 1] == currSymbol)) {

```

Comment:
Почему бы скобки сразу не раскрыть?
```suggestion
                        if (arrStr[i + 1] != currSymbol) {

```